universal-application-tool-0.0.1/app/assets/javascripts/preview.ts

```diff
@@ -98,6 +100,30 @@ class PreviewController {
       "Add " + entityType);
   }
 
+  /**
+   * Sets the child nodes of the selected div as text or span nodes.
+   * This will highlight text matching PreviewController.THIS_REGEX
+   * in span nodes.
+   *
+   * This will only work when the selected div is only supposed to contain
+   * text and has no other child nodes.
+   */
+  static setChildrenTextNodes(selector: string, text: string) {

```

Comment:
nit: `setChildrenTextNodes` is kinda vague. Maybe rename to `highlightEnumeratorReferences` or something.

Reply:
renamed to `setTextAndHighlightEnumeratorReferences`.